client/utils/reorder.js

```diff
@@ -0,0 +1,13 @@
+import { mapActions } from 'vuex-module';
```

Comment:
Please move it within outline folder and rename it to reorderMixin (unless you had more ideas where you could include it).